fs/fstestutil/record/record.go

```diff
@@ -1,11 +1,11 @@
-package record // import "bazil.org/fuse/fs/fstestutil/record"
+package record // import "github.com/suffuse/fuse/fs/fstestutil/record"
```

Comment:
The change to .travis.yml accompanies it because it is affected by it. The README I could have left but I don't overthink the commit history of documentation.